assets/src/scss/partials/country-selector-old.scss

```diff
@@ -0,0 +1,175 @@
+@import "../base/mixins";
+@import "../base/variables";
+@import "../base/colors";
+@import "../base/fonts";
+
+$menu-height-large: 3.75rem;
```

Comment:
It's not ideal to copy these here from the navbar, but it's the old design so we don't intend to change it and will remove soon.